library/core/src/macros/mod.rs

```diff
@@ -1364,7 +1364,7 @@ pub(crate) mod builtin {
         /* compiler built-in */
     }
 
-    /// Attribute macro applied to a static to register it as a global allocator.
+    /// Attribute macro applied to a static to register it as a global allocator. See also [`std::alloc::GlobalAlloc`](../std/alloc/trait.GlobalAlloc.html).
```

Comment:
Thanks for sending in the patch. I believe it would be better to wrap it is more readable for people with small terminal (some parts are 80 but it would be good to always keep it below 100).

```suggestion
    /// Attribute macro applied to a static to register it as a global allocator.
    /// See also [`std::alloc::GlobalAlloc`](../std/alloc/trait.GlobalAlloc.html).
```

Reply:
In order for only the first line to show up in the rustdoc summary, an extra newline between lines `1367` and `1368` is required too I believe. Otherwise the second string will be concatenated to the first in the summary.

## Screenshot
This is the summary view I'm talking about ([src](https://doc.rust-lang.org/core/index.html)).

![Screenshot_2020-09-26 core - Rust](https://user-images.githubusercontent.com/2467194/94344451-cf7d1780-001f-11eb-8c66-b1c5e2367b06.png)

Reply:
Resolved? By the way, regarding the strategy on adding subsequent commits to a PR I filed https://github.com/rust-lang/rustc-dev-guide/issues/900.